--- app/build.gradle.kts
+++ app/build.gradle.kts
@@ -24,20 +24,14 @@
   }
 
   signingConfigs {
-    create("debugConfig") {
-      storeFile = file("${rootDir}/debug.keystore")
-      storePassword = "android"
-      keyAlias = "androiddebugkey"
-      keyPassword = "android"
-    }
     create("release") {
       val keystorePath = System.getenv("KEYSTORE_PATH") ?: ""
       val keystoreFile = if (keystorePath.isNotBlank()) file(keystorePath) else null
       
       if (keystoreFile != null && keystoreFile.exists() && keystoreFile.isFile) {
         storeFile = keystoreFile
         storePassword = System.getenv("STORE_PASSWORD")
         keyAlias = System.getenv("KEY_ALIAS") ?: "upload"
         keyPassword = System.getenv("KEY_PASSWORD")
       }
     }
   }
 
   buildTypes {
     release {
       isCrunchPngs = false
       isMinifyEnabled = false
       proguardFiles(getDefaultProguardFile("proguard-android-optimize.txt"), "proguard-rules.pro")
       
       val keystorePath = System.getenv("KEYSTORE_PATH")
       if (!keystorePath.isNullOrBlank() && file(keystorePath).exists()) {
           signingConfig = signingConfigs.getByName("release")
       } else {
-          signingConfig = signingConfigs.getByName("debugConfig")
+          signingConfig = signingConfigs.getByName("debug")
       }
     }
     debug {
-      signingConfig = signingConfigs.getByName("debugConfig")
     }
   }
